eng/helixcorefxtests.proj

```diff
@@ -0,0 +1,111 @@
+<Project InitialTargets="BuildHelixWorkItems" Sdk="Microsoft.DotNet.Helix.Sdk">
+  <Import Project="$(NuGetPackageRoot)microsoft.dotnet.build.tasks.feed\$(MicrosoftDotNetBuildTasksFeedVersion)\build\Microsoft.DotNet.Build.Tasks.Feed.targets" />
+
+  <PropertyGroup>
+    <!-- Set the TargetFramework just to make the SDK happy -->
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Creator>$(_Creator)</Creator>
+    <HelixAccessToken>$(_HelixAccessToken)</HelixAccessToken>
+    <HelixBuild>$(_HelixBuild)</HelixBuild>
+    <HelixSource>$(_HelixSource)</HelixSource>
+    <HelixTargetQueues>$(_HelixTargetQueues)</HelixTargetQueues>
+    <HelixType>$(_HelixType)</HelixType>
+    <HelixArchitecture>$(__BuildArch)</HelixArchitecture>
+    <HelixConfiguration>$(__BuildType)</HelixConfiguration>
+
+    <!--
+      TODO: these should be moved to Directory.Build.props (from dir.props)
```

Comment:
I think this comment can be removed. It is not clear yet to me whether we will remove dir.props in favor of Directory.Build.props.

For the mid term we will keep both to allow building on either arcade or buildtools.

Reply:
Directory.Build.props is the "arcade" way/replace for explicitly including a "dir.props".  I added this comment because I felt that is the direction coreclr should go (to be more arcade "like", but still be backward compatible with buildtools). 

But if we are sure that Directory.Build.props isn't the way to go, I'll remove the comment.

FYI, including "..\dir.props" here made things really hard to get working because it "mixed" the two different build systems.  It would be nice (if possible), to have the two separate "tracks", one that is almost purely "arcade" for SDK projects and the old build system with the old props/targets, etc.

Reply:
Can you change the comment to:

```
<!--
      TODO: ProjectDir, RootBinDir, TestWorkingDir, and TargetsWindows are global properties set in dir.props, remove the property assignment here when we port to arcade.
```

Reply:
Will do.